src/pages/networks/CreateNetwork.tsx

```diff
@@ -117,6 +117,7 @@ const CreateNetwork: FC = () => {
         project={project}
         section={section}
         setSection={updateSection}
+        version={0}
```

Comment:
`version` prop could be optional so that we don't need to pass it in for `Create...` components?

Reply:
good idea, we can default it to 0